.circleci/config.yml

```diff
@@ -1,205 +1,21 @@
-          key: v1-dependencies-{{ checksum "requirements.txt" }}-{{ .Environment.CIRCLE_JOB }}
-          paths:
-            - env
-
-      - run: env/bin/coverage run -m unittest discover
-
-
-  test-osx-py37:
-    <<: *osx-tests-template
-    environment:
-      PYTHON: 3.7.0
-      HOMEBREW_NO_AUTO_UPDATE: 1
-
-  test-osx-py36:
-    <<: *osx-tests-template
-    environment:
-      PYTHON: 3.6.5
-      HOMEBREW_NO_AUTO_UPDATE: 1
-
-  test-osx-py35:
-    <<: *osx-tests-template
-    environment:
-      PYTHON: 3.5.5
-      HOMEBREW_NO_AUTO_UPDATE: 1
-      BASH_ENV: /Users/distiller/repo/.circleci/.bashrc_macOS
-
-  #
-  # Windows x64
-  #
-
-  test-win-py38: &win-tests-template
-    executor:
-      name: win/default
-
-    environment:
-      PYTHON: 3.8.0
-
-    working_directory: ~/repo
-
-    steps:
-      - checkout
-
-      # Install python
-      - restore_cache:
-          key: nuget-{{ .Environment.CIRCLE_JOB }}
-      - run:
-          name: install python
-          command: nuget install python -Version $env:PYTHON -ExcludeVersion -OutputDirectory .
-      - save_cache:
-          key: nuget-{{ .Environment.CIRCLE_JOB }}
-          paths:
-            - .\python
-
-      # Set up virtual environment
-      - run:
-          name: create virtualenv
-          command: |
-            .\python\tools\python.exe --version
-            .\python\tools\python.exe -m pip install virtualenv
-            .\python\tools\python.exe -m virtualenv env
-
-      # Install requirements
-      - restore_cache:
-          key: v1-dependencies-{{ checksum "requirements.txt" }}-{{ .Environment.CIRCLE_JOB }}
-      - run: &win-install-dependencies-template
-          name: install dependencies
-          command: |
-            env\Scripts\activate.ps1
-            python --version
-            pip install -r requirements.txt
-            pip install coverage
-      - save_cache:
-          key: v1-dependencies-{{ checksum "requirements.txt" }}-{{ .Environment.CIRCLE_JOB }}
-          paths:
-            - env
-
-      # Run tests
-      - run: &win-run-unittests-template
-          name: run unittests
-          command: |
-            env\Scripts\activate.ps1
-            coverage run -m unittest discover
-
-  test-win-py37:
-    <<: *win-tests-template
-    environment:
-      PYTHON: 3.7.5
-
-  test-win-py36:
-    <<: *win-tests-template
-    environment:
-      PYTHON: 3.6.8
-
-  test-win-py35:
-    <<: *win-tests-template
-    environment:
-      PYTHON: 3.5.4
+  dwave: dwave/orb-examples@1.0.0
 
 workflows:
   version: 2.1
   tests:
     jobs:
-      - test-py38
-      - test-py37
-      - test-py36
-      - test-py35
-      - test-osx-py38
-      - test-osx-py37
-      - test-osx-py36
-      - test-osx-py35
-      - test-win-py38
-      - test-win-py37
-      - test-win-py36
-      - test-win-py35
+      - dwave/test-linux:
+          matrix:
+            parameters:
+              python-tag: ["3.5", "3.6", "3.7", "3.8", "3.9"]
```

Comment:
I guess that even if we had these use a default specified in the orb, we would need to change the orb version number in order to use it? So there's no way to add say 3.10 at the orb level without updating all of the individual examples?

Reply:
Right...instead of passing in the python version to the job, we could just have the job run all of the versions.

Reply:
Will give that a try now.

Reply:
If we're going to be "pushing" new versions from the orb-level, we should also add a [nightly workflow](https://circleci.com/docs/2.0/workflows/#nightly-example). So that we can detect that something went wrong.

Reply:
Actually, from my understanding, I don't think we can have a job run all the versions. Each job has one executor (defining the environment). You also can't define workflows in orbs.

Reply:
I suppose we could define a single job that tests all version of python. Maybe a min version and max version with default specified or something like that. Would make it hard to use the docker containers the way we are... Might not be worth it, especially since we might not want to be changing the tested versions out from under the projects at all.

Reply:
There is a `latest` tag that can be applied to docker images. Maybe add a nightly job with that?